code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -102,7 +102,7 @@
 		for(var/overlay in atmos_overlay_types-new_overlay_types) //doesn't remove overlays that would only be added
 			vis_contents -= overlay
 
-	if (new_overlay_types.len)
+	if (length(new_overlay_types))
```

Comment:
length or len seems to more optimal depending on the patch, I'm starting to wonder if we should either not bother changing it or make a macro we can swap depending on how byond is feeling that week.